src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -19,6 +19,7 @@ import Conditions from './conditions';
 import Result from './result';
 import Intro from './intro';
 import EarlyAdopterMessage from './earlyAdopterMessage';
+import SavedQueries from './savedQueries';
```

Comment:
Are relative or absolute imports the way we should be doing things? I see a mix of both sometimes.

Reply:
We tend to use absolute imports with the exception of when it's in the same code path. Here I'm using absolute imports for everything except discover specific modules

Reply:
Thank you :+1: